weave-js/src/common/components/WBMenu.tsx

```diff
@@ -54,6 +52,25 @@ const DEFAULT_OPTION_RENDERER: OptionRenderer = ({
   </S.Item>
 );
 
+/**
```

Comment:
TSLint craps the bed with ES2023 so this is a polyfill for `findLastIndex`... not stoked on this solution though b/c we should be able to use baseline supported JS methods. I didn't want to remove the deprecated TSLint package in this PR though.

Reply:
**suggestion:** this seems like a common util. since it's being exported, is there another place we could define it that would be more appropriate?